sniffr/dog_routes/dog_routes.py

```diff
@@ -206,19 +164,20 @@ def post_dog(current_user):
 
 
 @dog_bp.route("/dog/<dog_id>", methods=["DELETE"])
-# @token_required
-def delete_dog(dog_id):
+@token_required
+def delete_dog(current_user, dog_id):
     """Create or edit dog info"""
+    user_id = int(current_user.user_id)
+    dog_id = int(dog_id)
 
-    # user_id = current_user.user_id
-    queried_dog = db.session.query(Dog).filter(Dog.dog_id == dog_id).first()#.filter(Dog.owner_id == user_id).first()
```

Comment:
Did you mean to leave in the #.filter at the end?

Reply:
That was in code that was removed :)